### PR TITLE
[@types/react-content-loader] introduce `primaryOpacity` and `secondaryOpacity` props + bump version to 3.1

### PR DESCRIPTION
**Pull request by samwalshnz**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://github.com/danilowoz/react-content-loader/pull/96>
- [x] Increase the version number in the header if appropriate.

### CODE REVIEW

File: types/react-content-loader/index.d.ts [hunk 1]
```diff
@@ -1,22 +1,44 @@
...
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.6
 
-import * as React from 'react';
+import * as React from "react";
 
 export interface ContentLoaderProps {
     style?: React.CSSProperties;
-    type?: 'facebook' | 'instagram' | 'list' | 'bullet-list' | 'code';
+    type?: "facebook" | "instagram" | "list" | "bullet-list" | "code";
```
Comment (hasantayyar) [inline types/react-content-loader/index.d.ts]: Why double quotes?

Comment (samwalshnz) [reply types/react-content-loader/index.d.ts]: Apologies - it looks like my code formatter has taken over

**Review by alaatm — CHANGES_REQUESTED** [review]
Can you use single quotes please since this repo uses single quotes.

**Comment by samwalshnz** [comment]
@alaatm I have fixed up those code formatting changes

**Review by alaatm — APPROVED** [review]

**Review by rbuckton — APPROVED** [review]

**Pull request merged**